### ISSUE TITLE
IOError on terminal happening often (kanes.retailarchitects.com/tg/order/query)

### ISSUE THREAD

**Issue by kbower** [kbower/tickettest1#554]
_Imported from trac ticket 554.
Created by **rneville**
Opened in trac: Jan 18, 2011, 10:54:17 AM
Last modified in trac: Jan 25, 2011, 4:51:19 PM_
***
_by rneville:_
```
Error message:
Error: [IOErrorEvent type="ioError" bubbles=false cancelable=false
eventPhase=2 text="Error #2032"]. URL:
https://kanes.retailarchitects.com/tg/order/query
```
Jerry @ 01
Reports a particular terminal (maybe) keeps getting IOError. Advised this is most likely a network error.

The IO error reports the problem is with the tg/order/query service.

This is being caused by row results for Oracle not limited (at all)...

**Comment by kbower** [comment]
_**rneville** trac status change on Jan 18, 2011, 10:54:26 AM:_ **closed**

**Comment by kbower** [comment]
_**rneville** trac status change on Jan 18, 2011, 11:04:25 AM:_ **reopened**

**Comment by kbower** [comment]
_**rneville** trac comment on Jan 18, 2011, 11:05:14 AM:_

Keep getting IOError on /tg/order/query in random spots in the application. My theory is that these are old requests that maybe timed out or something....not positive what to do about it yet.

**Comment by kbower** [comment]
_**rneville** trac comment on Jan 19, 2011, 11:05:54 AM:_

seems to have resolved itself

**Comment by kbower** [comment]
_**rneville** trac status change on Jan 19, 2011, 11:05:54 AM:_ **closed**

**Comment by kbower** [comment]
_**kbower** trac status change on Jan 25, 2011, 3:39:08 PM:_ **reopened**

**Comment by kbower** [comment]
_**kbower** trac comment on Jan 25, 2011, 4:51:19 PM:_

(In [[1124]](../commit/94da46b5cd3726ba7e02182b5fe6e59b8f76b6ff))
 * fix IOError on terminal happening often (fixes #554)
 * Recent orders on dashboard should restrict to site/user (closes [#557](557))

**Comment by kbower** [comment]
_**kbower** trac status change on Jan 25, 2011, 4:51:19 PM:_ **closed**
